api-reference/ingestion/lease/find-leases-for-tenant.mdx

```diff
@@ -0,0 +1,219 @@
+          The second line of the lease's address.
+        </ResponseField>
+        <ResponseField name="city" type="string">
+          The city of the lease's address.
+        </ResponseField>
+        <ResponseField name="country" type="string">
+          The country of the lease's address.
+        </ResponseField>
+        <ResponseField name="state" type="string">
+          The state in which the lease's address is located.
+        </ResponseField>
+        <ResponseField name="postal_code" type="string">
+          The postal code of the lease's address.
+        </ResponseField>
+        <ResponseField name="latitude" type="float">
+          The latitude of the lease's address.
+        </ResponseField>
+        <ResponseField name="longitude" type="float">
+          The longitude of the lease's address.
+        </ResponseField>
+      </Expandable>
+    </ResponseField>
+  </Expandable>
+</ResponseField>
+
+### Errors
+
+Listed below are common errors that may be returned by the endpoint, along with their corresponding status code.
+
+Status Code - 400
+```json Missing Authorization header
+{
+   "error": {
+    "type": "missing_authorization",
+    "message": "Your request does not include an 'Authorization' header with a bearer token for your account."
+  }
+}
+```
+
+Status Code - 401
+```json Invalid/Expired Authorization header bearer token value
+{
+   "error": {
+    "type": "invalid_authorization",
+    "message": "The bearer token you have provided in the 'Authorization' header is invalid."
+  }
+}
+```
+
+Status Code - 400
+
+Returned if none of the tenant parameters are provided in the request body
+```json No tenant information
+{
+   "error": {
+    "type": "no_tenant_information",
+    "message": "You need to provide at least one of identifier, email_address or phone_number in the request body."
+  }
+}
+```
+
+<RequestExample>
+
+```bash Request
+curl --location --request POST 'https://api.travtus.com/leases/find-for-tenant/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: bearer <token>' \
+--data-raw '{
+    "email_address": "test@example.com",
+    "active": true
+}'
+```
+
+</RequestExample>
+
+<ResponseExample>
+
+```json Response
+{
+  "leases": [
+    {
+      "identifier": "lease-1",
+      "start_date": "2023-01-17",
+      "end_date": "2023-12-17",
+      "move_in_date": "2023-01-17",
+      "landlord_id": "landlord-id-1",
+      "tenant_id": "tenant-id-1",
+      "property_id": "property-id-1",
+      "is_furnished": false,
+      "minimum_days_notice": 30,
+      "deposit": 1400.0,
+      "rent": 700.0,
+      "unit_number": "1",
+      "address": {
+        "building_name": "Building 1",
+        "address_line_1": "Blvd Street",
+        "address_line_2": "Appartment 12",
+        "city": "New York",
+        "country": "United States",
+        "state": "New York",
+        "postal_code": "10001",
+        "latitude": 40.730610,
+        "longitude": -73.935242
```

Comment:
Are these real coordiantes from prod?

Reply:
I am not 100% sure, but I got them by googling lat and long coordinates of New York :D https://www.latlong.net/place/new-york-city-ny-usa-1848.html
Do you think there's a good case for changing them to something else?

Reply:
I checked, they point to a generic place in Queens, NY. Doesn't look like it's anything PII related so should be fine